### PR TITLE
Doc: Clarify rules of reusing received messages

### PR DESCRIPTION
**Pull request by rbuch**

### CODE REVIEW

File: doc/charm++/manual.rst [hunk 1]
```diff
@@ -3646,12 +3646,12 @@ should *not* reuse the buffer. That is, after you have passed a message
...
 access its fields, or pass that same buffer into a second entry method
 invocation. Note that this rule doesn’t preclude the *single reuse* of
-an input message - consider an entry method invocation :math:`i_1`,
-which receives as input the message buffer :math:`m_1`. Then,
-:math:`m_1` may be passed to an asynchronous entry method invocation
-:math:`i_2`. However, once :math:`i_2` has been issued with :math:`m_1`
-as its input parameter, :math:`m_1` cannot be used in any further entry
-method invocations.
+a received message - consider being inside the body of an entry method
+invocation :math:`i_1`, which has receiveed the message buffer :math:`m_1`
```
Comment (matthiasdiener) [inline doc/charm++/manual.rst:3650]: received

**Comment by stwhite91** [comment]
Is 'CkCopyMsg' documented anywhere?

File: doc/charm++/manual.rst (hunk 1 above)
Comment (rbuch) [reply doc/charm++/manual.rst]: Thanks, fixed.

**Comment by rbuch** [comment]
> Is 'CkCopyMsg' documented anywhere?

Nope, not that I can find.

**Pull request merged**